ui/src/common/QualityInfo.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Chip from '@material-ui/core/Chip'
+
+const useStyles = makeStyles((theme) => ({}))
```

Comment:
Is it needed? Unless you add customization points to it (see the `spotify-ish` PR), I think this should be removed